src/AWS.Deploy.Orchestration/Utilities/DeployedApplicationQueryer.cs

```diff
@@ -124,16 +124,24 @@ public async Task<List<CloudApplication>> GetCompatibleApplications(List<Recomme
                 catch (FailedToUpdateLocalUserSettingsFileException ex)
                 {
                     _orchestratorInteractiveService.LogErrorMessageLine(ex.Message);
+                    if (session.Diagnostics)
```

Comment:
Again we should push the diagnostics check into the _orchestratorInteractiveService.

Reply:
Fixed